### PR TITLE
Add e2e test for metrics event

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: test/e2e/api_test.go [hunk 1]
```diff
@@ -47,6 +47,16 @@ func TestRegisterEvents(t *testing.T) {
 		User:      user,
 	})
 	assert.NoError(t, err)
+	iecMetricsEvt, err := json.Marshal(&api.InternalErrorCountMetricsEvent{
```
Comment (cre8ivejp) [inline test/e2e/api_test.go:50]: Please use a non-error case of metric events. Reporting errors to test the API could be misunderstood as real errors.

**Review by cre8ivejp — CHANGES_REQUESTED** [review]

File: test/e2e/api_test.go [hunk 2]
```diff
@@ -59,6 +73,11 @@ func TestRegisterEvents(t *testing.T) {
 				Event: goalEvent,
 				Type:  api.GoalEventType,
 			},
+			{
```
Comment (cre8ivejp) [inline test/e2e/api_test.go:76]: Please rebase this branch and use the `newEvent` that was introduced here.

https://github.com/ca-dp/bucketeer-go-server-sdk/pull/91/files#diff-40be257af42e6fcbb9b3cb9b1e41df34ac9d1b83ee2c32a91a4722389b196060R331

Comment (ono-max) [reply test/e2e/api_test.go:76]: I don't want to make `newEvent` func public for tests.

**Review by cre8ivejp — APPROVED** [review]
Thanks!

**Review by cre8ivejp — CHANGES_REQUESTED** [review]
Please fix the conflicts.

**Review by cre8ivejp — APPROVED** [review]
Thanks!

**Pull request merged**